CMakeLists.txt

```
@@ -189,6 +189,13 @@ set(DBUS_CONFIG_FILENAME        "org.freedesktop.DisplayManager.conf"
 set(COMPONENTS_TRANSLATION_DIR  "${DATA_INSTALL_DIR}/translations"                  CACHE PATH      "Components translations directory")
 set(SDDM_INITIAL_VT             "1"                                                 CACHE STRING    "Initial tty to use")
 
+# Intentionally does not use CMAKE_INSTAL_PREFIX, just like SessionDir in the Configuration.h
+set(DEFAULT_THEME_DIRS "/usr/local/share/sddm/themes" "/usr/share/sddm/themes")
```

I think we should try `/usr` first, and then `/var/lib/sddm/themes`:

```suggestion
set(DEFAULT_THEME_DIRS "${CMAKE_INSTALL_FULL_DATADIR}/sddm/themes" "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/lib/sddm/themes")
```